SnapMD.VirtualCare.ApiModels/Scheduling/AppointmentsRequest.cs

```diff
@@ -52,5 +52,15 @@ public class AppointmentsRequest : AvailabilityBlocksRequest
         /// The include patient co-userss.
         /// </value>
         public bool? IncludePatientCoUsers { get; set; }
+
+        /// <summary>
+        /// Records to skip.
+        /// </summary>
+        public int? Skip { get; set; }
```

Comment:
@nonsense66 In our current usage, do we put skip and take as properties of a class or as input parameters? Do they look different in the querystring or would they be written the same?